pydoctor/sphinx_ext/help_output.py

```diff
@@ -0,0 +1,38 @@
+"""
+Produce the pydoctor help output to be included in the documentation.
+"""
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+from contextlib import redirect_stdout
+from io import StringIO
+from pydoctor.driver import parse_args
+
+
+class HelpOutputDirective(Directive):
+    """
+    Directive that will generate the pydoctor help as block literal.
+    """
+    has_content = True
+
+    def run(self):
```

Comment:
Please add type annotations on new code. They do help a lot finding bugs and making maintenance safer.